src/Pim/Bundle/EnrichBundle/Elasticsearch/Sorter/InGroupSorter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Pim\Bundle\EnrichBundle\Elasticsearch\Sorter;
+
+use Pim\Bundle\CatalogBundle\Doctrine\ORM\Repository\GroupRepository;
+use Pim\Bundle\CatalogBundle\Elasticsearch\Sorter\Field\BaseFieldSorter;
+use Pim\Component\Catalog\Exception\InvalidArgumentException;
+use Pim\Component\Catalog\Query\Sorter\FieldSorterInterface;
+
+/**
+ * @author    Philippe Mossière <philippe.mossiere@akeneo.com>
```

Comment:
Maybe one thing you could add is a bit of explanation (where is it used and why ?)